eoscostarica/lifebank
# Feat/close account

Resolves #738 
Resolves #810 


### What does this PR do?
Enable users to closes their accounts

### How should this be tested?
- Make run
- Go to login
- Check reopen account modal
